helm/dagster/schema/schema_tests/test_dagster_daemon.py

```diff
@@ -247,3 +248,20 @@ def test_daemon_labels(template: HelmTemplate):
 
     assert set(deployment_labels.items()).issubset(daemon_deployment.metadata.labels.items())
     assert set(pod_labels.items()).issubset(daemon_deployment.spec.template.metadata.labels.items())
+
+
+def test_sensor_threading(instance_template: HelmTemplate):
```

Comment:
Can also test if the configuration has all the fields in the configurable class, similar to:

https://github.com/dagster-io/dagster/blob/b1aa9599f40b8655fc389996f42018ce7ac75e5a/helm/dagster/schema/schema_tests/test_instance.py#L471-L472